docs/core/whats-new/dotnet-7.md

```diff
@@ -0,0 +1,181 @@
+
+Also in the template configuration file, you can now annotate a template parameter as allowing multiple values. For example, the [`web` template](../tools/dotnet-new-sdk-templates.md) allows multiple forms of authentication.
+
+For more information, see the [.NET 7 Preview 6](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-6/#template-authoring) blog post.
+
+### Publish to a container
+
+Containers are one of the easiest ways to distribute and run a wide variety of applications and services in the cloud. Container images are now a supported output type of the .NET SDK, and you can create containerized versions of your applications using [`dotnet publish`](../tools/dotnet-publish.md). For more information about the feature, see [Announcing built-in container support for the .NET SDK](https://devblogs.microsoft.com/dotnet/announcing-builtin-container-support-for-the-dotnet-sdk/). For a tutorial, see [Containerize a .NET app with dotnet publish](../docker/publish-as-container.md).
+
+### Central package management
+
+You can now manage common dependencies in your projects from one location using NuGet's central package management (CPM) feature. To enable it, you add a *Directory.Packages.props* file to the root of your repository. In this file, set the MSBuild property `ManagePackageVersionsCentrally` to `true` and add versions for common package dependency using `PackageVersion` items. Then, in the individual project files, you can omit `Version` attributes from any [PackageReference](../project-sdk/msbuild-props.md#packagereference) items that refer to centrally managed packages.
+
+For more information, see [Central package management](/nuget/consume-packages/Central-Package-Management).
+
+## P/Invoke source generation
+
+.NET 7 introduces a source generator for platform invokes (P/Invokes) in C#. The source generator looks for <xref:System.Runtime.InteropServices.LibraryImportAttribute> on `static`, `partial` methods to trigger compile-time source generation of marshalling code. By generating the marshalling code at compile time, no IL stub needs to be generated at run time, as it does when using <xref:System.Runtime.InteropServices.DllImportAttribute>. The source generator improves application performance and also allows the app to be ahead-of-time (AOT) compiled. For more information, see [Source generation for platform invokes](../../standard/native-interop/pinvoke-source-generation.md) and [Use custom marshallers in source-generated P/Invokes](../../standard/native-interop/tutorial-custom-marshaller.md).
+
+## Related releases
+
+This section contains information about related products that have releases to coincide with the .NET 7 release.
```

Comment:
```suggestion
This section contains information about related products with releases that coincide with the .NET 7 release.
```

Reply:
I kept part of your suggestion, but I think "that have" is easier to translate than "with". With "with", the sentence could be taken to mean "This section contains info about related products. [This section] has releases to coincide..."